src/app/admin/adminApp.js

```diff
@@ -39,6 +55,170 @@ const authProvider = ({ signIn, signOut }) => ({
+          validate={[required()]}
+          label="Specialization / position"
+        /> */}
+        <CheckboxGroupInput
+          source="Specialization_position"
+          choices={[
+            {
+              id: "psychologicalConsultant",
+              name: "Psychological consultant",
+            },
+            { id: "psychotherapist", name: "Psychotherapist" },
+            { id: "psychiatrist", name: "Psychiatrist" },
+            { id: "sexologist", name: "Sexologist" },
+            { id: "SocialWorker", name: "Social worker" },
+          ]}
+          label="Specialization / position"
+          validate={required()}
+        />
+        <TextInput
+          source="full_name"
+          validate={[required()]}
+          style={{ width: "300px" }}
+          label="Full name"
+        />
+        <div>
+          <RadioButtonGroupInput
+            source="sex"
+            choices={[
+              { id: "female", name: "female" },
+              { id: "male", name: "male" },
+            ]}
+            validate={[required()]}
+          />
+        </div>
+        <div>
+          <NumberInput source="years_of_experience" min={0} />
+        </div>
+        <div>
+          <SelectInput
+            source="service_format"
+            choices={[
+              { id: "online", name: "online" },
+              { id: "offline", name: "offline" },
+              { id: "і online, і offline", name: " online, offline" },
+            ]}
+            validate={[required()]}
+            onChange={onServiceFormatChange}
+          />
+        </div>
+        {shouldShowAddressFields && (
+          <div>
+            <ArrayInput source="addresses">
+              <SimpleFormIterator inline>
+                <TextInput source="full_address" validate={[required()]} />
+                <TextInput source="clinic_name" />
+                <SelectInput
+                  source="district"
+                  choices={[
+                    { id: "Личаківський", name: "Личаківський" },
+                    { id: "Шевченківський", name: "Шевченківський" },
+                    { id: "Франківський", name: "Франківський" },
+                    { id: "Залізничний", name: "Залізничний" },
+                    { id: "Галицький", name: "Галицький" },
+                    { id: "Сихівський", name: "Сихівський" },
+                  ]}
+                  validate={[required()]}
+                />
+              </SimpleFormIterator>
+            </ArrayInput>
+          </div>
+        )}
+
+        <SelectArrayInput
+          source="type_of_therapy"
+          choices={[
+            { id: "admin", name: "Індивідуальна" },
+            { id: "u001", name: "Для дітей і підлітків" },
+            { id: "u002", name: "Сімейна" },
+            { id: "u003", name: "Групова" },
+            { id: "u004", name: "Для пар" },
+            { id: "u005", name: "Для бізнесу" },
+          ]}
+          validate={[required()]}
+        />
```

Comment:
in the requirements for this task, this field should be a multiple choice field, while requiring at least one choice